<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>741</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="CommonlLayout">
     <item>
      <layout class="QVBoxLayout" name="ServerLayout">
       <item>
        <widget class="QLabel" name="ServerLabel">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Server</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="RequestsLabel">
         <property name="text">
          <string>Requests</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="RequestsList"/>
       </item>
       <item>
        <widget class="QLabel" name="DataLabel">
         <property name="text">
          <string>Data</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="DataInput"/>
       </item>
       <item>
        <widget class="QLabel" name="EventLabel">
         <property name="text">
          <string>Event</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="EventInput"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="SendFromServerlLayout">
         <item>
          <spacer name="SendFromServerLeftSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>68</width>
             <height>28</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="SendFromServerButton">
           <property name="text">
            <string>Send</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="SendFromServerRightSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>68</width>
             <height>28</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="ClientLayout">
       <item>
        <widget class="QLabel" name="ClientLabel">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Client</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ResponsesEventsDataLabel">
         <property name="text">
          <string>Responses, events &amp; data</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="ResponsesEventsDataList"/>
       </item>
       <item>
        <widget class="QLabel" name="RequestLabel">
         <property name="text">
          <string>Request</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="RequestInput"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="SendRequestButtonLayout">
         <item>
          <spacer name="SendRequestButtonLeftSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>78</width>
             <height>28</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="SendRequestButton">
           <property name="text">
            <string>Send request</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="SendRequestButtonRightSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>78</width>
             <height>28</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>763</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
